client/src/hooks/useMembers.js

```diff
@@ -0,0 +1,97 @@
+import { useState, useEffect } from 'react'
+import { firebaseApp, MembersDbRef } from '../base'
+
+function useMembers() {
```

Comment:
Imho this hook is not necessary. Maybe refactor your databse methods into a `service.js` file. 
Also there is a danger of data loss! You export setMembers, but do NOT have any effects when members changes..